orion/pkg/repos/migrations/000005_create_table_locations.up.sql

```diff
@@ -1,15 +1,15 @@
 CREATE TABLE locations
 (
-    id           int unsigned NOT NULL AUTO_INCREMENT,
-    created_at   datetime     NOT NULL,
-    updated_at   datetime     NOT NULL,
-    deleted_at   datetime,
-    loc_id       varchar(64)  NOT NULL UNIQUE,
-    street       varchar(255) NOT NULL,
-    city         varchar(64)  NOT NULL,
-    state        varchar(64)  NOT NULL,
-    zipcode      varchar(64)  NOT NULL,
-    room         varchar(64),
+    id         int unsigned NOT NULL AUTO_INCREMENT,
+    created_at datetime     NOT NULL,
+    updated_at datetime     NOT NULL,
+    deleted_at datetime,
+    loc_id     varchar(64)  NOT NULL UNIQUE,
+    street     varchar(255) NOT NULL,
+    city       varchar(64)  NOT NULL,
+    state      varchar(64)  NOT NULL,
+    zipcode    varchar(64)  NOT NULL,
+    room       varchar(64),
```

Comment:
I've always wondered why these files always get changed. Looks like your IDE is automatically re-formatting these spaces?

Reply:
Yeah there was an extra space that my IDE removed.

Reply:
but then other people's IDEs are changing it back lol yikes

Reply:
I'm not sure why that file changed. The other ones stayed the same.